boards/arm/thingy91_nrf9160/board_nonsecure.c

```diff
@@ -7,8 +7,16 @@
 #include <init.h>
 
 #if defined(CONFIG_NRF_MODEM_LIB) && defined(CONFIG_NET_SOCKETS_OFFLOAD)
+#if defined(CONFIG_POSIX_API)
```

Comment:
These changes will be coming to NRF through another PR: https://github.com/nrfconnect/sdk-nrf/pull/4565

Reply:
I've removed this change from this PR